app/views/taxons/_common.html.erb

```diff
@@ -1,5 +1,9 @@
 <% content_for :title, presented_taxon.title %>
 <% content_for :meta_tags do %>
+  <% if !presented_taxon.live_taxon? %>
```

Comment:
This part of the change is from the PR to hide draft taxons from search engines. I think to be clear that we're reverting this as well, this change (and it's accompanying test) should either be separated out into another commit or the commit message for this commit should be updated.